### DISCUSSION TITLE
Raw Assembly Test

### DISCUSSION THREAD

**Discussion by giscus** [aduros/wasm4#49] [Games]
# Raw Assembly Test

Build retro games using WebAssembly for a fantasy console

https://wasm4.org/play/raw-assembly/?giscus=33971f8068426dfc9ccc6a2a1tB5Al3wQljZwP8FhY2VNve450at402eyZPYsZs4SQXczrJyT%2Fw8DtGcLGK6bAeNi%2FGFCB7VEEwth6oXof7nwR7uStbGB%2BkUx8VGuQRgg5z6Y5Fgb0nh%2BCcLHqM%3D

**Comment by aduros** [comment]
An experiment in hand-written assembly: https://github.com/aduros/wasm4/blob/main/examples/raw-assembly/src/cart.wat

**Reply by takaeri** [reply]
This should be renamed to "**Raw WebAssembly**" instead of "**Raw Assembly**"
